### PR TITLE
lang: Fuzzing host

### PR DESCRIPTION
**Pull request by armaniferrante**
Addresses https://github.com/project-serum/anchor/issues/123.

### CODE REVIEW

File: lang/fuzzing/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+edition = "2018"
+
+[dependencies]
+bincode = "1.3.1"
+bumpalo = { version = "3.4.0", features = ["collections", "boxed"] }
+lazy_static = "1.4.0"
+rand = "0.7.3"
+safe-transmute = "0.11.0"
+spl-token = { version = "3.0.1", features = ["no-entrypoint"] }
+solana-program = "=1.5.15"
```
Comment (armaniferrante) [inline lang/fuzzing/Cargo.toml:14]: TODO: update to solana 1.6.3

**Comment by macalinao** [comment]
@armaniferrante do you have any updates on this by any chance?

**Comment by armaniferrante** [comment]
> @armaniferrante do you have any updates on this by any chance?

No it's currently on hold.

**Comment by lengdavid** [comment]
@armaniferrante  Is this PR still on hold ?

**Pull request closed without merging**